crates/next-core/src/next_import_map.rs

```diff
@@ -236,9 +236,7 @@ pub fn get_next_client_resolved_map(
         // Temporary hack to replace the hot reloader until this is passable by props in next.js
         (
             context,
-            GlobVc::new(
-                "**/*/next/dist/client/components/react-dev-overlay/hot-reloader-client.js",
-            ),
+            GlobVc::new("**/next/dist/client/components/react-dev-overlay/hot-reloader-client.js"),
```

Comment:
locally rustfmt does this differently for me

Reply:
The CI checks fmt, so it's fine^^